.github/workflows/ci.yml

```diff
@@ -1,8 +1,9 @@
 name: CI
-on: [push, pull_request]
-
-env:
-  test-db-path: tests_ci/test_db.toml
+on:
```

Comment:
This will prevent running GA on feature branches for both this one and forks. If you intend to create PR's even for feature branches in progress, this may be not that big of a deal. We can add `workflow_dispatch` as well, so CI can be run on feature branches on request instead.

An alternative should be to leave it as is and change the concurrency to: `'${{ github.workflow }} @ ${{ github.ref }}'`. This should create a duplicate workflow but the second one should cancel the first one out. Not sure how this works out in practice.

Reply:
I generally create draft PRs for feature branches in progress, but I can definitely see the usefulness in having GA run prior to PR creation.

> An alternative should be to leave it as is and change the concurrency to: `'${{ github.workflow }} @ ${{ github.ref }}'`. This should create a duplicate workflow but the second one should cancel the first one out. Not sure how this works out in practice.

I'll give this a try. If it doesn't cause weird reports like (18 succeeded, 18 cancelled), I'm perfectly fine with doing it this way.

Reply:
Well that didn't work 🙈. Then your solution is fine I guess.

Reply:
Hmm, I'm not sure how to properly solve this. Duplicate jobs are never cancelled because `github.ref` resolves to either `refs/head/<branch>` or `refs/pull/<PR ID>/merge` depending on whether it was triggered by `push` or `pull_request`. `gitlab.sha` won't work either because for `pull_request`, it always refers to a merge commit.

It would be nice if the `pull_request` trigger would only happen if the PR came from a fork, but I'm not seeing an obvious way to do that.

Reply:
Exactly, I noticed the duplicate workflows too yesterday and went to look for a solution and this is one of the solutions that came up. However I noticed you were working on this as well so I didn't investigate any further. For now your proposed solution will do fine.

Reply:
Sounds good. I'll change it back to this for now:
```yaml
on: 
  push:
    branches:
      - master
  pull_request:
```

Reply:
For the sake of completeness: I managed to create a concurrency group which cancels one run:

```
${{ github.workflow }}-${{ github.event.pull_request.head.label || format('{0}:{1}', github.repository_owner, github.ref_name) }}
```

However it is still not that useful as it always shows 1 cancelled check (along with a notification) which will show that the checks have failed with a red cross.

So the current solution is better.